id-policy.md

```diff
@@ -0,0 +1,231 @@
+**http://purl.obolibrary.org/obo/GO_0050918** 
+
+## Policy for OBO Foundry ontologies
+
+All OBO ontologies should use PURLs that have a base URL within their allocated space (in CURIE form these would be prefixed with their ID space)
+
+
+## Response to Web requests for OBO URIs
+
+It is expected that the Foundry-compliant URIs behave, on the web,
+usefully. It will be the role of the OBO Foundry to supply generic
+software for responding to requests at URIs that identify OBO terms.
+
+
+We borrow the criteria from the Shared Name Initiative
+(http://sharedname.org/)as a base line. The OBO Foundry may issue
+further recommendations if experience shows them to be considered
+generally useful.
+
+
+
+1.  It must be clearly stated what the intended referent of each URI is
+    supposed to be, i.e. that the URI denotes some particular record
+    from some particular database.
+
+2.  Information about the URI and its referent, including such a
+    statement, must be made available, and in order to leverage existing
+    protocol stacks, it must be obtainable via HTTP. (We will call such
+    information "URI documentation".)
+
+3.  URI documentation must be provided in RDF.
+
+4.  Provision of URI documentation must be an ongoing concern. The
+    ability to provide it may have to outlive the original ontology
+    developer's group or creator.
+
+5.  The provider of the URI documentation must be responsive to
+    community needs, such as the need to have mistakes fixed in a timely
+    manner.
+
+6.  URI documentation must be open so that it can be replicated and
+    reused.
+
+
+
+Individual ontology projects may, at their discretion, choose to manage
+these responses, with the understanding that if service lapses the
+Foundry may substitute the generic software for handling them in order
+to maintain service.
+
+## Policy for OBO Library ontologies
+
+OBO Library ontologies are not constrained by this policy, however, we
+recommend that they follow it nonetheless, for three reasons. First, it
+provides a uniform experience and sets expectations for ontology
+clients. Second, by doing so library ontologies will be able to take
+advantage of shared infrastructure. Third, ontologies that eventually
+join the foundry would have to disrupt their ids if they had to change
+to follow this policy.
+
+**Allocating IDSPACEs**
+
+IDSPACEs within the OBO library are unique for a given project and are
+chosen not to conflict with prefix for xrefs. Although IDSPACEs are
+case-sensitive, there will never be more than one IDSPACEs that are the
+same when compared in a case-insensitive manner. Therefore,although
+"GO" and "go", "Go" and "gO" are different IDSPACEs, the IDSPACE "go",
+"Go" and "gO" will not be used as "GO" has already been allocated.
+
+
+
+A registry of allocated IDSPACEs will be maintained. Requests for an
+IDSPACE should be made by sending mail to
+obo-discuss@lists.sourceforge.net, cc obo-admin@fruitfly.org. A request
```

Comment:
wasn't there a new group email address? Though looking at the latest emails it seems everybody still uses the sf one.

Reply:
google groups